packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -1,54 +1,60 @@
-// @ts-check
-/* global setImmediate */
+// no @ts-check
```

Comment:
Once we flip from "opt in" to "opt out" defaults then the above absence of pragma would be checked. 
The way to opt out:
```suggestion
// @ts-nocheck
```

Reply:
My guess is you disabled it because `t.context` isn't typed. I'd suggest disabling just those errors so we don't lose type assistance on the rest of the file.

You could also type the context like https://github.com/Agoric/agoric-sdk/blob/d5d1b565f5803e8209425d04b9c0b9b244e42e36/packages/run-protocol/test/swingsetTests/vaultFactory/test-vaultFactory.js#L20-L23

Reply:
There's too many. It's too confused about the context, and I don't think it needs to be. I'll wait for some expert to straighten that out more generally.